kolibri/plugins/learn/assets/test/views/content-card-group-carousel.spec.js

```diff
@@ -125,13 +125,16 @@ describe('ContentCardGroupCarousel', function() {
           this.wrapper.setProps({contents});
           this.wrapper.instance().elSize.width = this.wrapper.instance().contentCardWidth * 3;
           // NOTE tried avoiding a nextTick in every test, didn't work. Dunno why.
-          // Vue.nextTick(done);
+          // return Vue.nextTick();
         });
         it('contains the first content item passed in', function() {
           const wrapper = this.wrapper;
           const carousel = wrapper.instance();
+          const contents = makeContentData(15);
```

Comment:
Doesn't work when I do everything the `beforeEach` is supposed to do within the test either :/